adi_update_boot.sh

```diff
@@ -476,9 +476,9 @@ restoring_extra_files()
         exit 1
       fi
       echo "u-boot.img restored."
-      echo -e "\nRestoring extlinux/extlinux.conf (specific to cyclone5 projects starting with 2021_r1 release)..."
+      echo -e "\nRestoring extlinux/extlinux.conf (specific to arria10 projects starting with 2021_r1 release)..."
```

Comment:
As Nuno said, I would add another commit for updating these echo's and also would be great to optimize it a bit: 
- lines 455-465 and 479-489 are doing the same thing: restoring extlinux.conf.
- also lines 442-451 and 466-477 are restoring u-boot (src for c5soc / img for a10soc).
You can create methods and just call them here.